issue_template.md

```diff
@@ -1,25 +1,43 @@
-* 1. What does this package do? (explain in 50 words or less)  
-* 2. Paste the full DESCRIPTION file inside a code block below.  
+### Summary
+
+- [ ] What does this package do? (explain in 50 words or less):
+
+
+- [ ] Paste the full DESCRIPTION file inside a code block below:
 
 ```
 
 ```
 
-* 3. URL for the package (the development repository, not a stylized html page)  
-* 4. What data source(s) does it work with (if applicable)?  
-* 5. Who is the target audience?  
-* 6. Are there other R packages that accomplish the same thing? If so, what is different about yours?  
-* 7. Check the box next to each policy below, confirming that you agree. These are mandatory.  
-    * [ ] This package does not violate the Terms of Service of any service it interacts with.  
-    * [ ] The repository has continuous integration with Travis CI and/or another service  
-    * [ ] The package contains a vignette  
-    * [ ] The package contains a reasonably complete README with instructions for installing the development version.  
-    * [ ] The package contains unit tests  
-    * [ ] The package only exports functions to the NAMESPACE that are intended for end users  
-* 8. Do you agree to follow the [rOpenSci packaging guidelines](https://github.com/ropensci/packaging_guide)? These aren't mandatory, but we strongly suggest you follow them. If you disagree with anything, please explain.  
-    * [ ] Are there any package dependencies not on CRAN?  
-    * [ ] Do you intend for this package to go on CRAN?  
-    * [ ] Does the package have a CRAN accepted license?    
-    * [ ] Did `R CMD check` (or `devtools::check()`) produce any errors or warnings? If so paste them below.  
-* 9. Please add explanations below for any exceptions to the above:  
-* 10. If this is a resubmission following rejection, please explain the change in circumstances.  
+- [ ] URL for the package (the development repository, not a stylized html page):
+
+- [ ] Who is the target audience?  
+
+- [ ] Are there other R packages that accomplish the same thing? If so, what is different about yours?  
+
+#### Requirements
+
+Confirm each of the following by checking the box.  This package:
+
+- [ ] does not violate the Terms of Service of any service it interacts with. 
+- [ ] only exports functions to the NAMESPACE that are intended for end users
+- [ ] has a test suite
+- [ ] has continuous integration with Travis CI and/or another service  
+- [ ] contains a vignette with examples of its essential functions and uses
+- [ ] contains a README with instructions for installing the development version. 
+- [ ] has a CRAN and OSI accepted license.
+
+#### Publication options
+
+- [ ] Do you intend for this package to go on CRAN?  
+- [ ] Do you wish to automatically submit to the [Journal of Open Source Software](http://joss.theoj.org/)?
+- [ ] The package contains a [`paper.md`](http://joss.theoj.org/about#paper_structure) with a high-level description.
```

Comment:
Should these be nested options? It seems like these are only relevant if the author checks the 'Do you wish to automatically submit to JOSS'?